runtime/help/commands.md

```diff
@@ -31,9 +31,15 @@ quotes here but these are not necessary when entering the command in micro.
 
 * `quit`: quits micro.
 
-* `goto 'line'`: jumps to the given line number. A negative number can be
-   passed to jump inward from the end of the file; for example, -5 jumps
-   to the 5th-last line in the file.
+* `goto '[-+]?line'(:'[-+]?col')?`: Jumps to the given line (and column) number.
```

Comment:
`[-+~]`?
Or maybe it's better not to specify the precise syntax here, just simple `'line[:color]'`, since you document the prefixes anyway.

Reply:
Lol, I meant `'line[:col]'`.